### PR TITLE
tt: add page

### PR DESCRIPTION
**Pull request by RemoteRabbit**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** v0.4.2

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages/linux/tt.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> A terminal based typing tester.
+> More information: <https://github.com/lemnos/tt>.
+
+- Start quote mode with builting quote list in english 'en':
+
+`tt -quotes en`
+
+- Produce a test consisting of 50 randomly drawn words in 5 groups of 10 words each:
+
+`tt -n 10 -g 5`
```
Comment (blueskyson) [inline pages/linux/tt.md:12]:
```suggestion
`tt -n {{10}} -g {{5}}`
```

File: pages/linux/tt.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Start a timed test lasting 10 seconds:
+
+`tt -t 10`
```
Comment (blueskyson) [inline pages/linux/tt.md:16]:
```suggestion
`tt -t {{10}}`
```

File: pages/linux/tt.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# tt
+
+> A terminal based typing tester.
```
Comment (acuteenvy) [inline pages/linux/tt.md:3]:
```suggestion
> A terminal based typing test.
```

File: pages/linux/tt.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://github.com/lemnos/tt>.
+
+- Start quote mode with builting quote list in english 'en':
```
Comment (acuteenvy) [inline pages/linux/tt.md:6]:
```suggestion
- Start quote mode with the builtin quote list in English:
```

File: pages/linux/tt.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`tt -quotes en`
```
Comment (acuteenvy) [inline pages/linux/tt.md:8]:
```suggestion
`tt -quotes {{en}}`
```

File: pages/linux/tt.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Produce a test consisting of 50 randomly drawn words in 5 groups of 10 words each:
+
+`tt -n 10 -g 5`
+
+- Start a timed test lasting 10 seconds:
+
+`tt -t 10`
+
+- Start tt with no theming and showing your WPM as you type:
```
Comment (acuteenvy) [inline pages/linux/tt.md:18]:
```suggestion
- Start `tt` with no theming and showing your WPM as you type:
```

File: pages/linux/tt.md (hunk 1 above)
Comment (RemoteRabbit) [reply pages/linux/tt.md]: Updated

File: pages/linux/tt.md (hunk 3 above)
Comment (RemoteRabbit) [reply pages/linux/tt.md]: Updated

File: pages/linux/tt.md (hunk 5 above)
Comment (RemoteRabbit) [reply pages/linux/tt.md]: Updated

File: pages/linux/tt.md (hunk 6 above)
Comment (RemoteRabbit) [reply pages/linux/tt.md]: Updated

File: pages/linux/tt.md (hunk 4 above)
Comment (RemoteRabbit) [reply pages/linux/tt.md]: Updated

File: pages/linux/tt.md (hunk 2 above)
Comment (RemoteRabbit) [reply pages/linux/tt.md]: Updated

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**